### ISSUE TITLE
[bench request]: Add local-jev (Qwen3.5-4B zero-shot, no fine-tuning; TypeSafe wire format)

### ISSUE THREAD

**Issue by amithgc** [fstandhartinger/jevbench#15]
Request to add [local-jev](https://github.com/amithgc/local-jev) (MIT code; zero-shot `Qwen/Qwen3.5-4B`, Apache-2.0 weights, no fine-tuning and no adapter) to the ranked systems.

**What it is.** A local server that speaks TypeSafe's `/v1/systemone` wire format, so your `typesafe` adapter works unchanged (checked with the unmodified `typesafe-sdk`). The submitted model, `llm-qwen3.5-4b`, is frozen Qwen3.5-4B read at the answer token: each question and its options are laid out as one JSON object (SemIf's layout, credited), the options are lettered, and the answer is the softmax over the next-token logits of the letters. There is no generation. The state is prefilled once and its key-value cache is shared across the questions about it. Per-primitive temperatures are fitted on held-back rows of public, human-labelled datasets, never on JevBench items or on any Jev output.

```sh
git clone https://github.com/amithgc/local-jev && cd local-jev && git checkout 64a0b31
uv venv --python 3.12 .venv && uv pip install --python .venv/bin/python -e .
.venv/bin/local-jev serve --model llm-qwen3.5-4b --host 0.0.0.0 --port 8765
# first start downloads Qwen/Qwen3.5-4B (~9.3 GB) and loads the text model only (~8.5 GB, fp16 on a GPU);
# the model answers throwaway questions at load, so the first timed request is warm.
# no API key is needed; to require one, start it with LOCAL_JEV_API_KEY=<key> and pass Authorization: Bearer <key>.

python -m jevbench.cli run --tasks datasets/public/original.jsonl --adapter typesafe \
  --endpoint http://127.0.0.1:8765 --key-env '' --model llm-qwen3.5-4b ...
```

**Our own numbers on the public items** (your harness and `typesafe` adapter, one request at a time, Apple M4 Max, PyTorch on the Mac GPU):

| tier | items | llm-qwen3.5-4b |
|---|---|---|
| easy | 48 | 1.000 |
| standard (original) | 72 | 0.958 |
| hard | 111 | 0.622 |

Overall 0.805 on the 231 public items; median latency 651 ms, p95 6.9 s on that Mac. Serving has not been tested on CUDA by us, and PyTorch has no fast path for Qwen3.5's linear-attention layers on Apple GPUs, so latency on your hardware may differ a lot. We are not claiming a rank: the held-out items, calibration, speed and cost are yours to measure.

**Disclosure.** No JevBench item, paraphrase or Jev output was used for training, calibration or data generation; the model is not trained at all. The JSON-versus-plain prompt layout was chosen by measuring both on the 231 public items (plain 0.732, JSON 0.805), so the public numbers are mildly optimistic, and the held-out items may score lower.

Happy to answer questions here or rerun anything.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by fstandhartinger** [comment]
Congratulations on the JevBench v1.4 release, Amith Chandrappa (amithgc)! Thanks for building and sharing your system.

- local-jev Qwen3.5-4B: #10 at 46.80

local-jev Qwen3.5-4B is newly public in v1.4. V1.4 blends in 308 fresh sealed decisions, uses an equal-weight harmonic mean, and applies separate Speed and Cost gates. For your leading row, sealed accuracy was 26.0% versus 80.5% on the public slice. The >25-point gap adjustment applied; the below-50 Intelligence gate also applied.

Full board and method: https://benchmarkheaven.com/jev-models
